packages/api/package.json

```diff
@@ -7,7 +7,8 @@
   "scripts": {
     "build": "tsc",
     "lint": "eslint . --ext .js,.ts",
-    "prebuild": "rm -rf dist/ @types/",
+    "prebuild": "rm -rf dist/ @types/; npm run prebuild.packageConfig",
+    "prebuild.packageConfig": "node -p \"'export const PACKAGE_NAME = \\'' + require('./package.json').name + '\\';\\nexport const PACKAGE_VERSION = \\'' + require('./package.json').version + '\\';'\" > src/package.ts",
```

Comment:
Had to add this build step for our package file because we were loading `../package.json` in a few files and TS was transpiling the dist directory into the following:

![screen_shot_2022-02-08_at_5 58 08_pm](https://user-images.githubusercontent.com/33762/153288947-d695a5ab-4020-40ec-905e-cb71c2bfb2c2.png)